strangerstudios/paid-memberships-pro
# Increases Stripe Application Fees

### All Submissions:






### Changes proposed in this Pull Request:

- New installation will make use of the 2% Stripe fee
- Existing installations that update to the latest version will continue using the 1% Stripe fee

### How to test the changes in this Pull Request:

1. New Installation: Install and activate the plugin. Run a test purchase through Stripe Connect and check the fees. 2% should be charged.
2. Existing Installation: Updae the plugin so that the DB version is updated. Run a test purchase through Stripe Connect and check the fees. 1% should be charged.

### Other information:




### Changelog entry

Stripe application fee changes
